zinit-install.zsh

```diff
@@ -1451,6 +1454,32 @@ builtin source "${ZINIT[BIN_DIR]}/zinit-side.zsh" || {
 
     return $?
 } # ]]]
+# FUNCTION: .zinit-single-line [[[
+# Display cURL progress bar on a single line
+.zinit-single-line() {
```

Comment:
@psprint Does this seem reasonable? I am moving this code from its own file (i.e., `share/single-line.zsh`) into a function with the rest of the Zinit code.

Reply:
I'm not sure, if it works then I guess it's fine.